src/components/throw-panel/throwPanel.jsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, Flex } from "@chakra-ui/react"
+import { useState } from "react"
+import "./throwPanel.css"
+
+export function ThrowPanel () {
+
+    const [result, setResult] = useState([])
+    
+    function handleDiceThrow (a, b) {
+        let newThrow 
+            if (a === 1) {
+                newThrow = Math.floor(Math.random() * (b - 1 + 1) + 1)
```

Comment:
интересно откуда - 1 + 1 появилось? Свиду кажется что их можно сложить сразу и получится 0 т.е. вот так? 
```suggestion
                newThrow = Math.floor(Math.random() * b + 1)
```

Reply:
Исправил, убрал лишние единицы